src/utils/commandRunner.ts

```diff
@@ -32,10 +32,10 @@ export class CommandRunner {
     let executablePath: string;
     let command: string;
     let runEnv: NodeJS.ProcessEnv | undefined;
-    const isEEEnabled = this.settings.executionEnvironment.enabled;
+    const isEEEnabled = this.settings.executionEnvironment?.enabled ?? false;
     const interpreterPath = isEEEnabled
       ? 'python3'
-      : this.settings.python.interpreterPath;
+      : this.settings.python?.interpreterPath;
```

Comment:
Why is relaxing that accessor need? If it really is, shouldn't we also prepare for it like this:
```suggestion
      : this.settings.python?.interpreterPath ?? 'python3';
```

Same applies to line 53